test/docs/__snapshots__/get-docs.api-test.ts.snap

```diff
@@ -18,7 +58,21 @@ components:
       type: apiKey
       in: header
       name: x-api-key
-  schemas: {}
+  schemas:
+    GetSiteStatusByExporterNameResponse:
+      type: object
+      properties:
+        siteId:
+          type: string
+          readOnly: true
+          example: '12345678'
+        status:
+          type: string
+          readOnly: true
+          example: Created
```

Comment:
Should this not have been an enum?

Reply:
I thought for tfs, if we were simply passing through the enum, we would do so as a string?

Reply:
(Alex and I had a chat about this: for GET responses in TFS, we didn't use an enum because we couldn't guarantee that the data in ACBS would follow that enum. For estore Alex checked that we can)